packages/react-components/src/queries/useHistoricalAssetPropertyValues/types.ts

```diff
@@ -13,10 +13,15 @@ export type QueryFnClient = {
   batchGetAssetPropertyValueHistory?: BatchGetAssetPropertyValueHistory;
 };
 
+export type FetchMode =
+  | 'START_TO_END'
+  | 'MOST_RECENT_BEFORE_START'
```

Comment:
Why do we need both most recent before start and end?

Reply:
I wanted to include this so that it supported the same features as the existing datasource implementation. We don't need it for alarms though.